glean-core/ios/Glean/GleanFfi.h

```diff
@@ -86,6 +86,64 @@ typedef const int32_t *RawIntArray;
 
 typedef const char *const *RawStringArray;
 
+/**
+ * ByteBuffer is a struct that represents an array of bytes to be sent over the FFI boundaries.
+ * There are several cases when you might want to use this, but the primary one for us
+ * is for returning protobuf-encoded data to Swift and Java. The type is currently rather
+ * limited (implementing almost no functionality), however in the future it may be
+ * more expanded.
+ *
+ * ## Caveats
+ *
+ * Note that the order of the fields is `len` (an i64) then `data` (a `*mut u8`), getting
+ * this wrong on the other side of the FFI will cause memory corruption and crashes.
+ * `i64` is used for the length instead of `u64` and `usize` because JNA has interop
+ * issues with both these types.
+ *
+ * ByteBuffer does not implement Drop. This is intentional. Memory passed into it will
+ * be leaked if it is not explicitly destroyed by calling [`ByteBuffer::destroy`]. This
+ * is because in the future, we may allow it's use for passing data into Rust code.
+ * ByteBuffer assuming ownership of the data would make this a problem.
+ *
+ * Note that alling `destroy` manually is not typically needed or recommended,
```

Comment:
```suggestion
 * Note that calling `destroy` manually is not typically needed or recommended,
```

Reply:
These are in the generated headers and are coming from ffi-support.
We shouldn't fix that on our end, but rather upstream: https://docs.rs/ffi-support/0.4.0/ffi_support/struct.ByteBuffer.html